systers_portal/meetup/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from meetup.views import MeetupLocationDetailView
+
+
+urlpatterns = [
+    url(r'^(?P<slug>\w+)/(?P<page_slug>\w+)/$',
```

Comment:
The first slug is for identifying meetup location and second page_slug is for what user need to see for that meetup location. 
For Eg: if url is - /meetup/boston/about that means user want to see about of meetup location with slug boston.
Basically first slug is to get row from database and second page_slug is  to get column of that row.